src/Symfony/Component/Routing/Attribute/Route.php

```diff
@@ -46,6 +46,7 @@ public function __construct(
         private array $requirements = [],
         private array $options = [],
         private array $defaults = [],
+        private array $alias = [],
```

Comment:
Perhaps add a phpdoc that its an array of string?
And make some validation preset?

Reply:
Done for the phpdoc.

What do you have in mind for validation ?

Reply:
by validation I meant just asserting that each array element is really a string, to not crash on `->addAlias` method